.editorconfig

```diff
@@ -0,0 +1,16 @@
+# EditorConfig <https://EditorConfig.org>
+root = true
+
+# elementary defaults
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = tab
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
```

Comment:
I don't know if we should keep `max_line_length`, since apparently it was removed from the EditorConfig specification.

Reply:
You're totally right